.github/workflows/test.yml

```diff
@@ -72,11 +72,17 @@ jobs:
       - id: rust-version
         run: "echo ::set-output name=version::$(rustc --version)"
 
-      - name: "Generate FFI"
+      - name: "Generate profiling FFI"
+        shell: bash
+        run: |
+           chmod +x build-profiling-ffi.sh
+           ./build-profiling-ffi.sh ${OUTPUT_FOLDER}/profiling
+
+      - name: "Generate Telemetry FFI"
         shell: bash
         run: |
            chmod +x build-profiling-ffi.sh
-           ./build-profiling-ffi.sh ${OUTPUT_FOLDER}
+           ./build-profiling-ffi.sh ${OUTPUT_FOLDER}/telemetry
```

Comment:
Should this be `build-telemetry-ffi`?

Reply:
Ohhh thats why the CI worked when the script was broken :facepalm: